### DISCUSSION TITLE
Can we define a different blocks for basalt type?

### DISCUSSION THREAD

**Discussion by DemonSlayer112** [null2264/CobbleGen#11] [Q&A] [answered]
You know how for basalt generation we need soul soil and blue ice?
Can we be able to change these two blocks?
Would be awesome if i could replace blue_ice for a water cauldron :D
this way we can have a 5gt cobble/stone farm in the nether!

tho just generally being able to change any of these two blocks feel powerful if we can do this, if not can it be added?

**Comment by null2264** [comment, marked as answer]
Unfortunately, it's not yet possible to do that. But it is a planned feature, I haven't added it because I ran into a couple of technical issues when testing it.

**Reply by null2264** [reply]
Update, I made a WIP build for this: https://github.com/null2264/CobbleGen/pull/23
